tests/scripts/depends.py

```diff
@@ -118,34 +125,38 @@ def restore_config(options):
     else:
         shutil.copy(options.config_backup, options.config)
 
-def run_config_py(options, args):
-    """Run scripts/config.py with the specified arguments."""
-    cmd = ['scripts/config.py']
-    if options.config != 'include/mbedtls/config.h':
-        cmd += ['--file', options.config]
-    cmd += args
-    log_command(cmd)
-    subprocess.check_call(cmd)
+def option_exists(conf, option):
+    return option in conf.settings
+
+def set_config_option_value(conf, option, colors, value: Union[bool, str]):
+    """Set/unset a configuration option, optionally specifying a value.
+value can be either True/False (set/unset config option), or a string,
+which will make a symbol defined with a certain value."""
+    if not option_exists(conf, option):
```

Comment:
`depends.py` in 2.28 currently relies on being able to unset an option that doesn't exist.

```
depends.py: Symbol MBEDTLS_SHA256_USE_A64_CRYPTO_IF_PRESENT was not found in include/mbedtls/config.h
```
I think it would be best to remove the A64 symbols from the 2.28 version of the file.

Reply:
Another good side of reusing well prepared scripts :)